### PR TITLE
Add more Inspektor Gadget talks

### PR DESCRIPTION
**Pull request by marga-kinvolk**
Recent talks were missing, this adds a few to the list. I'm sure I'm missing some others, but that's what I could find.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ Join the discussions on the [`#inspektor-gadget`](https://kubernetes.slack.com/m
 
 - Inspektor Gadget and traceloop, [FOSDEM 2020 - Brussels](https://fosdem.org/2020/schedule/event/containers_bpf_tracing/)
 - Traceloop for systemd and Kubernetes + Inspektor Gadget, [All Systems Go 2019 - Berlin](https://cfp.all-systems-go.io/ASG2019/talk/98A9LW/)
+- Using Inspektor Gadget with OpenShift, [Openshift Commons 2020](https://www.youtube.com/watch?v=X9PI7OWLJSY)
+- Using Inspektor Gadget and kubectl-trace, [Open Source Summit EU 2020](https://www.youtube.com/watch?v=X9PI7OWLJSY) (live version of the [Cloud Native BPF workshop](https://github.com/kinvolk/cloud-native-bpf-workshop))
```
Comment (mauriciovasquezbernal) [inline README.md:93]: This link is wrong, it's the same as above).

**Review by mauriciovasquezbernal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marga-kinvolk) [reply README.md:93]: Oops, you're right, copy & paste error, thanks for catching it!

**Review by mauriciovasquezbernal — COMMENTED** [review]
LGTM now!

**Comment by mauriciovasquezbernal** [comment]
@marga-kinvolk please squash the commits before merging!  :)

**Comment by marga-kinvolk** [comment]
Oops, sorry about the noise, I used an old copy of IG and messed up the rebase. Will redo :-/

**Review by mauriciovasquezbernal — APPROVED** [review]
Explicitly approving to unblock merge.

**Pull request merged**